packages/app/src/__deprecated__/renderer/components/core/modal/modal.component.tsx

```diff
@@ -50,7 +50,7 @@ export const ButtonContainer = styled.div<{ buttonsPosition: ModalSize }>`
   ${({ buttonsPosition }) => getButtonsPosition(buttonsPosition)};
   display: flex;
   box-sizing: border-box;
-  padding-top: 4rem;
+  padding-top: 2.4rem;
```

Comment:
Are you sure that it is needed? I'm afraid that this change can be shared with other modals that use this core.